tests/web/html/all-tests.html

```diff
@@ -14,6 +14,7 @@
         "../../../gpii/node_modules/matchMakerFramework/test/html/CanopyMatchMakerTest.html",
         "../../../gpii/node_modules/matchMakerFramework/test/html/MatchMakerUtilitiesTest.html",
         "../../../gpii/node_modules/transformer/test/html/TransformerTests.html",
+        "../../../gpii/node_modules/journal/test/html/JournalIdParserTests.html",
         "../../../gpii/node_modules/lifecycleManager/test/html/LifecycleManagerTest.html",
```

Comment:
We need to also add:

`../../../gpii/node_modules/lifecycleManager/test/html/DynamicComponentIndexerTest.html`